### ISSUE TITLE
PHP8.0: Can't install ext-ds

### ISSUE THREAD

**Issue by icanhazstring** [shivammathur/setup-php#318]
**Describe the bug**
Using PHP8.0 `ext-ds` can't be installed an I will receive.

```
✗ ds Could not install ds on PHP 8.0.0-dev
```

Using the local `Dockerfile` the extension gets installed, so there should not be any issue installing the extension.

```
FROM php:8.0.0RC2-alpine

COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/bin/
RUN install-php-extensions ds xdebug opcache
```

**Version**
- [x] I have checked releases, and the bug exists in the latest patch version of `v1` or `v2`.
- [x] `v2`
- [ ] `v1`

**Runners**

- [x] GitHub Hosted
- [ ] Self Hosted

**Operating systems**
ubuntu-latest

**PHP versions**
8.0

**To Reproduce**
```
unit-tests:
    name: "Unit Tests"
    runs-on: "${{ matrix.os }}"
    continue-on-error: "${{ matrix.experimental }}"
    strategy:
      fail-fast: false
      matrix:
        php: ["7.3", "7.4", "8.0"]
        os: ["ubuntu-latest"]
        experimental: [false]
        include:
          - php: "8.0"
            composerArgs: "--ignore-platform-req=php"

    steps:
      - name: Checkout
        uses: actions/checkout@v2

      - name: Setup PHP
        uses: shivammathur/setup-php@v2
        with:
          php-version: ${{ matrix.php }}
          tools: composer:v2, pecl
          extensions: ds
```

**Expected behavior**
`ext-ds` should be installed

**Screenshots/Logs**
![image](https://user-images.githubusercontent.com/883543/97424048-73ecc500-1910-11eb-86b8-d536d7d6bfcc.png)

**Comment by shivammathur** [comment]
@icanhazstring Fixed in 5d27b8f, updated `2.7.0` and `v2`.
Test workflow after fix - https://github.com/shivammathur/test-setup-php/runs/1322070476?check_suite_focus=true#step:3:14

**Comment by icanhazstring** [comment]
Nice. Thanks for quick fix ❤️
